packages/x-charts/src/LineChart/LineChart.tsx

```diff
@@ -454,6 +494,10 @@ LineChart.propTypes = {
       valueFormatter: PropTypes.func,
     }),
   ),
+  /**
+   * The configuration of the y axes.
+   * If not provided, a default axis config is used with id `DEFAULT_Y_AXIS_KEY`.
+   */
```

Comment:
```suggestion
  /**
   * The configuration of the y-axes.
   * If not provided, a default axis config is used with constant `DEFAULT_Y_AXIS_KEY`.
   */
```

Reply:
```suggestion
  /**
   * The configuration of the y-axes.
   * If not provided, a default axis config is used with id set to `DEFAULT_Y_AXIS_KEY`.
   */
```

Reply:
`DEFAULT_Y_AXIS_KEY` is not exactly the config. It's the id of the default config which is adapt depending on the chart you're using